README.md

```diff
@@ -1,4 +1,4 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This Waldo project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). And will have Express.
```

Comment:
Great job with your first commit!

We can also make this README have the project title like so:

```md
# Waldo

This Waldo project was bootstrapped...
```